creedengopython.properties

```diff
@@ -1,12 +1,19 @@
 category=External Analysers
 description=Provide a list of static code analyzers (for Python language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
 homepageUrl=https://github.com/green-code-initiative/creedengo-python
-publicVersions=2.0.0
-archivedVersions=
+publicVersions=2.0.1
+archivedVersions=2.0.0
 
 defaults.mavenGroupId=org.green-code-initiative
 defaults.mavenArtifactId=creedengo-python-plugin
 
+2.0.1.description=improve library versions; check and upgrades for SonarQube until 25.3.0 version
+2.0.1.sqcb=[24.12,25.3.*]
```

Comment:
I urge you to end the `sqcb` and `sqs` ranges with `LATEST`, altho this is up to you

Reply:
done